### DISCUSSION TITLE
Calling BlueprintNativeEvent-functions from C++ child class

### DISCUSSION THREAD

**Discussion by kevytmaito11** [Sixze/ALS-Refactored#265] [Q&A]
Is there a way to call BlueprintNativeEvent-functions from child C++ classes without creating implementation for them?

f. ex OnLocomotionModeChanged-event I added implementation to get event to child class:

	UFUNCTION(BlueprintNativeEvent, Category = "Als Character")
	void OnLocomotionModeChanged(const FGameplayTag& PreviousLocomotionMode);
	virtual void OnLocomotionModeChanged_Implementation(const FGameplayTag& PreviousLocomotionMode);

I can call these events from blueprint, but I could not find a way to call them from C++ child class if I did not create this implementation and override it.

**Comment by ameaninglessname** [comment]
In c++, just call `OnLocomotionModeChanged` should be enough

**Reply by kevytmaito11** [reply]
But if I try to create create definition to OnLocomotionModeChanged  in C++ child class it says that inherited member is not allowed. This only works in parent class.
